lib/guardian/user_guardian.rb

```diff
@@ -179,4 +179,9 @@ def can_upload_user_card_background?(user)
   def can_delete_sso_record?(user)
     SiteSetting.enable_discourse_connect && user && is_admin?
   end
+
+  def can_change_tracking_preferences?(user)
+    can_edit_user?(user) &&
+      (!user.staged || SiteSetting.allow_changing_staged_user_tracking)
```

Comment:
```suggestion
    (SiteSetting.allow_changing_staged_user_tracking || !user.staged) && can_edit_user?(user)
```

I flipped the conditions around based on which is cheaper to call.